### PR TITLE
[v0.1] Basic Count Statistics

### PR DESCRIPTION
**Pull request by robsonndarocha**

### CODE REVIEW

**Comment by jfarmer** [comment]
When creating a pull request for review, submit it against your own repository, not mine.

So, rather than asking to merge your changes into `jfarmer/text-analysis:master`, merge your changed into `robsonndarocha/text-analysis:master`.

I'll leave feedback here, though.

File: textalyze.js [hunk 1]
```diff
@@ -13,9 +13,21 @@
  */
 function itemCounts(array) {
   const counts = new Map();
+  var element = "";
+  var count = 0;
 
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
+  for(var i = 0; i < array.length; i++) {
```
Comment (jfarmer) [inline textalyze.js:21]: Right now you're looping over `array` once for every element in `array`.  Can you loop over it only once?

Comment (robsonndarocha) [reply textalyze.js:21]: I'm gonna try that as soon as I finish the iteration I'm on at the moment.

**Comment by robsonndarocha** [comment]
By the way...
I fixed the Pull Requests mistake I made. I created them on my own repo and invited you to be a collaborator. I'm gonna close all these Pull Requests I created here.

**Pull request closed without merging**